projects/ngrx.io/content/navigation.json

```diff
@@ -606,11 +590,6 @@
           "title": "Discord Server",
           "tooltip": "NgRx Community Discord Server - chat with other members of the community."
         },
-        {
-          "url": "https://gitter.im/ngrx/platform",
```

Comment:
I also removed this because there isn't much activity.
![image](https://github.com/ngrx/platform/assets/28659384/10ad1d2e-a801-469a-b1ec-c9f5249af453)
![image](https://github.com/ngrx/platform/assets/28659384/e56641c9-a35e-4dcc-8599-5f2c2634e00b)